presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/PrestoExceptionClassifier.java

```diff
@@ -205,6 +211,17 @@ public static boolean isClusterConnectionException(Throwable t)
         return Optional.empty();
     }
 
+    private static Optional<Throwable> getRequestThrottleExceptionCause(Throwable t)
+    {
+        while (t != null) {
+            if (t.getClass().equals(RuntimeException.class) && t.getMessage() != null && t.getMessage().contains("Request throttled")) {
```

Comment:
What about being a bit more flexible and resilient to future changes?
```
t instanceof RuntimeException
```
Also, where is `ClientException`? Is `t` not going to be a ClientException?

Reply:
Yes t is not going to be ClientException. It's going to be RuntimeException thrown by PrestoStatement#internalExecute method wrapped under SqlException